packages/terra-badge/src/Badge.scss

```diff
@@ -1,6 +1,24 @@
+@import '~terra-mixins';
 @import './variables';
 @import './mixins';
 
+@include cerner-mock-theme {
+  --terra-badge-background-color-default: #2c3e50;
+  --terra-badge-color-default: #fff;
+  --terra-badge-background-color-primary: #3498db;
+  --terra-badge-color-primary: #fff;
+  --terra-badge-background-color-secondary: #1abc9c;
+  --terra-badge-color-secondary: #fff;
+  --terra-badge-background-color-positive: #27ae60;
+  --terra-badge-color-positive: #fff;
+  --terra-badge-background-color-negative: #c0392b;
+  --terra-badge-color-negative: #fff;
+  --terra-badge-background-color-warning: #e67e22;
+  --terra-badge-color-warning: #fff;
+  --terra-badge-background-color-info: #bdc3c7;
+  --terra-badge-color-info: #333;
+}
+
 :local {
   /* stylelint-disable selector-class-pattern  */
   .badge {
```

Comment:
Actually, thinking about this... we'd only need that property on base.css. So the theme would only need to be defined in that one spot (and would turn the whole doc site gray).